### PR TITLE
Feature/readme update

### PR DESCRIPTION
**Pull request by bluebill1049**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+             }}
+           >
+           <YourComponent> // your component here
+        </Animate>;
+    }
+
+## API
+
+    startAnimation: boolean
+Default value as false, set true when animation starts.
```
Comment (yusinto) [inline README.md]: Defaults to false. Set to true to start the animation.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
...
+Child component to be animated
+
+    startStyle: string
+Component initial inline style
+
+    endStyle: string
+Component transition to inline style
+
+    onCompleteStyle: string (optional)
+Style applies after animation completed
```
Comment (yusinto) [inline README.md]: Style to be applied after the animation is completed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
...
+    $ npm install react-simple-animation
+
+## Quick start
+
+    import react from 'react';
+    import animate from 'react-simple-animate'
+    
+    export default function SexyComponent() {
+	    return <Animate durationSeconds={0.2}
+             startAnimation={startAnimation}
```
Comment (yusinto) [inline README.md]: Maybe set to true here so it works when users copy this example

**Comment by bluebill1049** [comment]
@yusinto thanks heaps yus

**Pull request merged**